tests/aws/services/kms/test_kms.validation.json

```diff
@@ -1,158 +1,242 @@
 {
+  "tests/aws/services/kms/test_kms.py::TestKMS::test_all_types_of_key_id_can_be_used_for_encryption": {
+    "last_validated_date": "2024-01-24T10:44:16+00:00"
+  },
+  "tests/aws/services/kms/test_kms.py::TestKMS::test_cant_delete_deleted_key": {
+    "last_validated_date": "2024-01-24T10:44:28+00:00"
+  },
+  "tests/aws/services/kms/test_kms.py::TestKMS::test_cant_use_disabled_or_deleted_keys": {
+    "last_validated_date": "2024-01-24T10:44:27+00:00"
+  },
   "tests/aws/services/kms/test_kms.py::TestKMS::test_create_alias": {
-    "last_validated_date": "2023-04-13T09:29:27+00:00"
+    "last_validated_date": "2024-01-24T10:43:41+00:00"
+  },
+  "tests/aws/services/kms/test_kms.py::TestKMS::test_create_grant_with_invalid_key": {
+    "last_validated_date": "2024-01-24T10:43:47+00:00"
+  },
+  "tests/aws/services/kms/test_kms.py::TestKMS::test_create_grant_with_same_name_two_keys": {
+    "last_validated_date": "2024-01-24T10:43:48+00:00"
+  },
+  "tests/aws/services/kms/test_kms.py::TestKMS::test_create_grant_with_valid_key": {
+    "last_validated_date": "2024-01-24T10:43:47+00:00"
   },
   "tests/aws/services/kms/test_kms.py::TestKMS::test_create_key": {
-    "last_validated_date": "2023-04-13T09:29:30+00:00"
```

Comment:
Similar to changes suggested for updating the snapshot tests, we can skip this for the scope of this PR.